db/lrs_views.sql

```diff
@@ -0,0 +1,164 @@
+
+drop view gewaesser_v;

```

Comment:
Q: Falls diese View "von uns" stammt wäre wohl `drop view if exists gewaesser_v;` korrekter, umn nicht in ein Bootstrap-Problem zu laufen (dies gilt für alle `drop`-Statements)